pom.xml

```diff
@@ -36,14 +36,14 @@
     <version.hdrHistogram>2.1.9</version.hdrHistogram>
     <version.javax.enterprise.concurrent-api>1.0</version.javax.enterprise.concurrent-api>
     <version.javax.interceptor-api>1.2.2</version.javax.interceptor-api>
-    <version.mp-fault-tolerance>2.0</version.mp-fault-tolerance>
+    <version.mp-fault-tolerance>2.1-SNAPSHOT</version.mp-fault-tolerance>
     <version.microprofile-config-api>1.3</version.microprofile-config-api>
-    <version.microprofile-metrics-api>1.1.1</version.microprofile-metrics-api>
+    <version.microprofile-metrics-api>2.0</version.microprofile-metrics-api>
     <version.slf4j-simple>1.7.26</version.slf4j-simple>
     <!-- smallrye-config is purely used for testing -->
     <version.smallrye-config>1.3.7</version.smallrye-config>
     <!-- smallrye-metrics is purely used for testing -->
-    <version.smallrye-metrics>1.1.3</version.smallrye-metrics>
+    <version.smallrye-metrics-2.0>1.0.0</version.smallrye-metrics-2.0>
```

Comment:
This needs to be 1.1.0 now as it was released today

Reply:
Good! I wanted to converge on SmallRye Metrics 2.0-1.1.0 anyway, since I hoped it would have eager metrics initialization and thread safety fixes (and indeed it does have both).